release-notes/major10/10.5.0/valtimo-frontend-libraries.md

```diff
@@ -43,6 +43,19 @@ The following features were added:
   For each Verzoek type under a Verzoek plugin configuration, a mapping can now be optionally configured. This specifies
   which fields should be copied from the Verzoek object to the document, instead of copying the entire object.
 
+* **Form.io translations enabled**
+
+  Valtimo's form.io components didn't utilize the built-in translation functionality for labels and content. By enabling this feature, translations can now be added to an implementation.
```

Comment:
```suggestion
  Valtimo's form.io components did not utilize the built-in translation functionality for labels and content. By enabling this feature, translations can now be added to an implementation.
```

Reply:
I did not get any email notification so totally forgot about this. I update pull request and fixed the merge conflicts